scripts.js

```diff
@@ -1 +1,45 @@
-console.log('Hello World!');
\ No newline at end of file
+// First homework task
+function getGreeting(name, surname) {
+    return (`Hi ${name} ${surname}! Good to see you`)
+}
+const greeting = getGreeting('John', 'Smith');
+console.log(greeting);
+// Second homework task
+function isThisMyName(name) {
+    return !!(name.includes('Szymon'))
```

Comment:
Why did you used double `!`? `String.includes` [is returning  true](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes#return_value)

Besides that is I pass `Szymonejro` the function would return true, but is should not. My suggestion is to simply use equality operator